Mathlib/Probability/StrongLaw.lean

```diff
@@ -641,7 +641,7 @@ end StrongLawAeReal
 
 section StrongLawVectorSpace
 
-variable {Ω : Type*} {mΩ : MeasurableSpace Ω} {μ : Measure Ω} [IsProbabilityMeasure μ]
+variable {Ω : Type*} {mΩ : MeasurableSpace Ω} {μ : Measure Ω} [IsZeroOrProbabilityMeasure μ]
```

Comment:
```suggestion
variable {Ω : Type*} {mΩ : MeasurableSpace Ω} {μ : Measure Ω} [IsProbabilityMeasure μ]
```
I would just keep this one: most of the proof is done assuming that the measure is a probability measure, but then the final result is deduced for an arbitrary measure. So there's no point in weakening the assumption in this section (unless confusing the readers).